packages/core/src/Practical.tsx

```diff
@@ -42,6 +42,15 @@ interface ToastsOptions {
   readonly defaultDurations: SimpleToastsDurations
 }
 
+/**
+ * User defined toast options. Similar to ToastsOptions but optional.
+ */
+export interface UserToastsOptions {
+  readonly placement?: ToastsPlacement
+  readonly alwaysOnTop?: boolean
+  readonly defaultDurations?: SimpleToastsDurations
+}
+
 const DEFAULT_TOASTS_OPTIONS: ToastsOptions = {
```

Comment:
export the `ToastsOptions` interface instead, and mark the properties as optional. Then the `DEFAULT_TOASTS_OPTIONS` can have the type `Required<ToastsOptions>`

Reply:
Done